tests/e2e/testfiles/authorization-templates/storage_csm_authorization_secret_provider_class.yaml

```diff
@@ -14,12 +14,17 @@ spec:
           key: username
         - objectName: dbRedisPassword
           key: password
+    - secretName: jwt-secret
+      type: Opaque
+      data:
+        - objectName: jwtsigningsecret
+          key: secret
   parameters:
     # Vault role name to use during login
     roleName: "csm-authorization"
-    vaultAddress: "https://vault0.default.svc.cluster.local:8400"
+    vaultAddress: "https://vault.default.svc.cluster.local:8400"
```

Comment:
The vaultSetupAutomation in run-e2e-test.sh names vault as vault0. This should stay at vault0